skyportal/handlers/api/source.py

```diff
@@ -3448,3 +3449,127 @@ async def get(self, obj_id):
             data = io.BytesIO(buf.read())
 
             await self.send_file(data, filename, output_type=output_format)
+
+
+class SourceCopyPhotometryHandler(BaseHandler):
+    @permissions(["Upload data"])
+    def post(self, objId):
+        """
+        ---
+        description: Copy photometry from one source to another
+        tags:
+          - notifications
+        requestBody:
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  objId:
+                    type: string
+                    description: |
+                      The ID of the Source's Obj the photometry is being copied to
+                  groupIds:
+                    type: array
+                    items:
+                      type: integer
+                    description: |
+                      List of IDs of groups to give photometry access to
+                  duplicateId:
+                    type: string
+                    description: |
+                      The ID of the Source's Obj the photometry is being copied from
```

Comment:
Maybe "originID" and "targetID" are better? also, it looks like you are getting the obj_id from the path. Should that be in the docstring?

Reply:
Good points.